cmd/deployment/create.go

```diff
@@ -200,6 +206,10 @@ func init() {
 	createCmd.Flags().String("enterprise-search-ref-id", "main-enterprise_search", "Optional RefId for the Enterprise Search deployment")
 	createCmd.Flags().Int32("enterprise-search-zones", 1, "Number of zones the Enterprise Search instances will span")
 	createCmd.Flags().Int32("enterprise-search-size", 4096, "Memory (RAM) in MB that each of the Enterprise Search instances will have")
+
+	// Remove in the next version.
+	createCmd.Flags().Bool("dt-as-list", true, "")
```

Comment:
In this case, instead of having a flag and hiding it. might it not make it simpler to assign `true` directly to `DeploymentTemplateAsList`?

Reply:
I did this so that we could keep the normal (Get) flow in the tests and toggle the `List()` with the flag.